docs/code-snippets/office-snippets.yaml

```
@@ -3499,6 +3545,32 @@ Office.TableData#rows:member:
         tableData.rows = [['row1'], ['row2'], ['row3']];
         return tableData;
     }
+Office.UI#displayDialogAsync:member(1):
+  - |-
+    // The following example shows how to open a dialog with a specified size. It also shows
+    // how to register a function to handle the message when Office.UI.messageParent() is called
+    // in the dialog. The implementation of the processMessage() function is omitted.
+
+    Office.context.ui.displayDialogAsync('https://www.contoso.com/myDialog.html', {height: 30, width: 20},
+        function (asyncResult) {
```

```suggestion
        (asyncResult) => {
```
To be consistent with format used in succeeding code.